codenney/fashion_api
# Ch create database migration

### What does this PR do?
* To create a migration file to create and populate the items and users table

### Description of the task to be completed?
* Install the dependencies

### How should this be manually tested?
* On the terminal, run the command `db-migrate up initialize` and `db-migrate down` to create the tables in the database if not exist or drop them.
* If successful, your should get a processed and done response

### Required
* Install the dependencies before attempting by running npm install from the terminal
* If needed, you can add extra data to the table(s)
